plugins/hls-hectare-plugin/hls-hectare-plugin.cabal

```diff
@@ -0,0 +1,30 @@
+cabal-version:      2.2
+name:               hls-hectare-plugin
+version:            1.0
+synopsis:           Code synthesis for typed holes via ECTA
+description:
+  Please see the README on GitHub at <https://github.com/haskell/haskell-language-server#readme>
+license:            Apache-2.0
+license-file:       LICENSE
+author:             Pepe Iborra
+maintainer:         pepeiborra@gmail.com
+category:           Development
+build-type:         Simple
+extra-source-files:
+  LICENSE
+
+library
+  if impl(ghc >= 8.10) && impl(ghc < 9.0)
```

Comment:
Given that we might drop support for 8.10.7 after the next major release, do we want to add plugins that only work on 8.10?

Reply:
Hectare itself should work for any GHC version, 
@Tritlo do you plan to upgrade the GHC plugin?  
cc @jkoppel